src/components/ActionBar/SourceVerificationIcon/SourceVerificationIcon.jsx

```diff
@@ -34,7 +34,7 @@ function SourceVerificationIcon({context, source, proxyUrl}) {
 
     function verify() {
         setNeedsVerification(true);
-        verifyFunction(sourceUrl, context.fetch).then((result) => {
+        verifyFunction(sourceUrl, context.underlyingFetchFunction()).then((result) => {
```

Comment:
This is wrong. Remove () after underlyingFetchFunction.
You forward the function return value here, but you should forward the function reference.
Check the behaviour by clicking the verified question mark -> it now returns failed but because of above error.

Reply:
fixed in 27f3e2be0417f34b48f75706d4707b4ae6cc5663